test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -101,6 +104,37 @@ public class HttpConnectionDispatcherTests
             }
         }
 
+        [Theory]
+        [InlineData(TransportType.WebSockets, "/ws", 404)]
+        [InlineData(TransportType.ServerSentEvents, "/sse", 404)]
```

Comment:
Add tests for the transports that **are** supported? Our other tests don't set `SupportedTransports`, I think, so we don't test that scenario currently.